editor/editor_tests/user_stats_test.cpp

```diff
@@ -2,19 +2,56 @@
 
 #include "editor/user_stats.hpp"
 
+#include "platform/platform_tests_support/writable_dir_changer.hpp"
+
 namespace editor
 {
 namespace
 {
-UNIT_TEST(UserStats_Smoke)
+auto constexpr kEditorTestDir = "editor-tests";
+WritableDirChanger wdc(kEditorTestDir, SettingsDirPolicy::UseWritableDir);
```

Comment:
Do you really want to change dir for all tests in editor_tests? I guess it's a bad idea.